crates/uv-cli/src/lib.rs

```diff
@@ -4334,6 +4334,10 @@ pub struct ToolListArgs {
     #[arg(long)]
     pub show_version_specifiers: bool,
 
+    /// Whether to display the extra requirements(s) installed with each tool.
```

Comment:
We might want to say "additional" here (as discussed elsewhere). If you want to update that language all at once, it can happen here or in a separate pull requests.

Reply:
I think it's also fine to just say "requirements" without the floating (s)

Reply:
```suggestion
    /// Whether to display the additional requirements installed with each tool.
```

(will require generated reference update)